Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -615,6 +627,28 @@ protected override void RenderContents (HtmlTextWriter writer)
         menu.RenderAsContextMenu(writer);
     }
 
+    private void RenderAriaTreeAlertBox (HtmlTextWriter writer)
+    {
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaLive, HtmlAriaLiveAttributeValue.Assertive);
+      writer.RenderBeginTag(HtmlTextWriterTag.Span);
+      writer.AddAttribute(HtmlTextWriterAttribute2.AriaHidden, HtmlAriaHiddenAttributeValue.True);
+      writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassScreenReaderText + " AriaTreeAlerts");
+      writer.RenderBeginTag(HtmlTextWriterTag.Span);
+      writer.Write("&nbsp;");
```

Comment:
why the nbsp?

Reply:
Thought I needed it, but it seems I didn't even know what it did, just thought that because we had it in some other areas, it would make sense here.